base_geoengine/geo_model.py

```diff
@@ -41,8 +41,10 @@ def fields_get(self, allfields=None, attributes=None):
 
     @api.model
     def _get_geo_view(self):
-        geo_view = self.env["ir.ui.view"].search(
-            [("model", "=", self._name), ("type", "=", "geoengine")], limit=1
+        geo_view = (
+            self.sudo()
+            .env["ir.ui.view"]
+            .search([("model", "=", self._name), ("type", "=", "geoengine")], limit=1)
         )
         if not geo_view:
             raise except_orm(
```

Comment:
except_orm is deprecated. Replace by UserError.

Reply:
I don't mind including the change, but I feel like that should be a different PR.